CharacterGenerator/CharacterGenerator/Manager.m

```diff
@@ -0,0 +1,38 @@
+//
+//  Manager.m
+//  CharacterGenerator
+//
+//  Created by Z on 8/6/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Manager.h"
+
+@implementation Manager
+
+//singleton code
++ (instancetype)sharedInstance
+{
+    static Manager *sharedInstance = nil;
```

Comment:
Nice use of a singleton! It would be a good exercise to try and implement this using the delegation pattern between the `SoldierListTableViewController` and the `CreateSoldierTableViewController'. Delegation would help you avoid having a model object that can be mutated from anywhere (as is the case with a singleton).